shell/platform/darwin/macos/framework/Headers/FlutterEngine.h

```diff
@@ -89,9 +91,40 @@ FLUTTER_DARWIN_EXPORT
  *
  * Setting this field from non-nil to a different non-nil FlutterViewController
  * is prohibited and will throw an assertion error.
+ *
+ * This method is deprecated. Querying view controllers should use
```

Comment:
Consider moving this snippet further up in the doc? I have to read about all the things this can do before I learn that I shouldn't use it.

Reply:
Also, there is no official annotation to mark something as deprecated that tools will pick up on?

Reply:
We have a macro for this. Added a suggestion below.